### PR TITLE
Initial/rest controllers generators

### PR DESCRIPTION
**Pull request by alexandervlasenko**

### CODE REVIEW

**Review by ILoveMyTortoise — APPROVED** [review]

File: src/Nox.Generator/Presentation/Api/ApiGenerator.Rest/ApiControllerGenerator.cs
Comment (rochar) [inline src/Nox.Generator/Presentation/Api/ApiGenerator.Rest/ApiControllerGenerator.cs:1]: @alexandervlasenko can we add a test to see if the class is generated, similar to what we have?

Comment (rochar) [reply src/Nox.Generator/Presentation/Api/ApiGenerator.Rest/ApiControllerGenerator.cs:1]: Can you provide a screen shoot of the generated class, so its easier to review, thks

**Review by rochar — CHANGES_REQUESTED** [review]
looks good, just a comment on tests and be able to see the generated class

File: src/Nox.Generator/Presentation/Api/ApiGenerator.Rest/ApiControllerGenerator.cs
Comment (alexandervlasenko) [reply src/Nox.Generator/Presentation/Api/ApiGenerator.Rest/ApiControllerGenerator.cs:1]: Hi @rochar , I've added a new test and also a sample file into examples folder (src/SampleWebApp/Examples/CountriesApiController.cs) so no need in a screenshot.

Best regards

**Review by rochar — APPROVED** [review]

**Review by jan-schutte — APPROVED** [review]

**Pull request merged**